src/onpage.js

```diff
@@ -18,9 +19,8 @@
 		scriptTag.src = 'SCRIPT_URL_HERE';
 		firstScript = doc.getElementsByTagName(scriptStr)[0];
 		firstScript.parentNode.insertBefore(scriptTag, firstScript);
-
-		root[branchStr] = branchSdk;
 	}
+	else { console.log("Branch is already embedded" ); }
```

Comment:
honestly i think it's fine if we fail silently, since they'll just be execuitng functions on an already-defined brnach object